tests/scripts/thread-cert/Cert_6_1_01_RouterAttach.py

```diff
@@ -60,10 +70,55 @@ def test(self):
         self.nodes[LEADER].start()
         self.nodes[LEADER].set_state('leader')
         self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
+        time.sleep(4)
 
         self.nodes[ED].start()
         time.sleep(5)
         self.assertEqual(self.nodes[ED].get_state(), 'child')
 
+        leader_messages = self.sniffer.get_messages_sent_by(LEADER)
+        ed_messages = self.sniffer.get_messages_sent_by(ED)      
+
+        # 1 - leader
+        msg = leader_messages.next_mle_message(mle.CommandType.ADVERTISEMENT)
+
+        # 2 - ed
+        msg = ed_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
+        msg.assertSentWithHopLimit(255)
+        msg.assertSentToDestinationAddress("ff02::2")
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Challenge)
+        msg.assertMleMessageContainsTlv(mle.ScanMask)
+        msg.assertMleMessageContainsTlv(mle.Version)
+
+        scan_mask_tlv = msg.get_mle_message_tlv(mle.ScanMask)
+        self.assertEqual(1, scan_mask_tlv.router)
+        self.assertEqual(0, scan_mask_tlv.end_device)
+
+        # 3 - leader
+        msg = leader_messages.next_mle_message(mle.CommandType.PARENT_RESPONSE)
+        msg.assertSentToNode(self.nodes[ED])
+
+        # 4 - ed
+        msg = ed_messages.next_mle_message(mle.CommandType.CHILD_ID_REQUEST)
+        msg.assertSentToNode(self.nodes[LEADER])
+        msg.assertMleMessageContainsTlv(mle.AddressRegistration)
+        msg.assertMleMessageContainsTlv(mle.LinkLayerFrameCounter)
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Response)
+        msg.assertMleMessageContainsTlv(mle.Timeout)
+        msg.assertMleMessageContainsTlv(mle.TlvRequest)
+        msg.assertMleMessageContainsTlv(mle.Version)
+        msg.assertMleMessageContainsOptionalTlv(mle.MleFrameCounter)
```

Comment:
`mle.MleFrameCounter` should be included in `ChildIdRequest` in current OT implementation. However the log shows 
`MleMessage doesn't contain optional TLV: <class 'mle.MleFrameCounter'>`

Please fix the method`assertMleMessageContainsOptionalTlv()` in message.py to output appropriate log.